src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -252,6 +253,22 @@ public  NewUserCreationRequest createNewUserRequest(String email) {
         return userCreationRequest;
     }
 
+    public  NewUserCreationRequest createReInviteUserRequest(String email) {
+        List<String> userRoles = new ArrayList<>();
+        userRoles.add("pui-user-manager");
+
+        NewUserCreationRequest userCreationRequest = aNewUserCreationRequest()
+                .firstName("someName")
+                .lastName("someLastName")
+                .email(email == RANDOM_EMAIL ? randomAlphabetic(10) + "@hotmail.com".toLowerCase() : email)

```

Comment:
/ let's see what happen when we compare String for equality using // == operator, it will return true if both String point to same // object e.g. interned String or String literal, otherwise it // will return false, even if two String contains same characters // and has same length

Please use equals/equals ignore case.

Reply:
Corrected